src/app/layout/header.component.html

```diff
@@ -5,19 +5,19 @@
     <!-- <a class="item" [ngClass]="{ 'active':currentRoute==='/datasets' }" routerLink="/datasets" routerLinkActive="active">Data Catalog</a> -->
     <!-- <a class="item" [ngClass]="{ 'active':currentRoute==='/environments' }" routerLink="/environments" routerLinkActive="active">Compute Environments</a> -->
     <div class="right menu">
-      <a class="item" [href]="docUrl" title="Users' Guide" target="_blank">
+        <a class="item" [href]="docUrl" title="Users' Guide" target="_blank" matTooltip="Open user's guide">
         <i class="info circle white icon"></i>
       </a>
-      <a class="item" href="https://github.com/whole-tale/whole-tale/issues/new/choose" title="Report a Problem" target="_blank">
+      <a class="item" href="https://github.com/whole-tale/whole-tale/issues/new/choose" title="Report a Problem" target="_blank" matTooltip="Report a problem">
```

Comment:
```suggestion
      <a class="item" href="https://github.com/whole-tale/whole-tale/issues/new/choose" target="_blank" matTooltip="Report a problem">
```

Removed `title` (basic HTML tooltip)